diff --git a/applications/luci-app-statistics/root/etc/config/luci_statistics b/applications/luci-app-statistics/root/etc/config/luci_statistics
index 74456720..772e2a89 100644
--- a/applications/luci-app-statistics/root/etc/config/luci_statistics
+++ b/applications/luci-app-statistics/root/etc/config/luci_statistics
@@ -23,7 +23,7 @@ config statistics 'collectd_rrdtool'
 	option RRARows '288'
 	option RRASingle '1'
 	option RRATimespans '2hour 1day 1week 1month 1year'
-	option backup '0'
+	option backup '1'
 
 config statistics 'collectd_csv'
 	option enable '0'
@@ -68,7 +68,7 @@ config statistics 'collectd_chrony'
 	option Timeout '2'
 
 config statistics 'collectd_conntrack'
-	option enable '0'
+	option enable '1'
 
 config statistics 'collectd_contextswitch'
 	option enable '0'
@@ -81,7 +81,8 @@ config statistics 'collectd_cpu'
 	option ValuesPercentage '1'
 
 config statistics 'collectd_cpufreq'
-	option enable '0'
+	option enable '1'
+	option ExtraItems '1'
 
 config statistics 'collectd_curl'
 	option enable '0'
@@ -115,10 +116,10 @@ config statistics 'collectd_exec'
 	option enable '0'
 
 #example for sqm_collect plugin
-#config collectd_exec_input
-#        option cmduser 'nobody'
-#        option cmdgroup 'nogroup'
-#        option cmdline '/usr/libexec/collectd/sqm_collectd.sh eth0 ifb4eth0'
+config collectd_exec_input
+        option cmduser 'nobody'
+        option cmdgroup 'nogroup'
+        option cmdline '/usr/libexec/collectd/sqm_collectd.sh eth0.2'
 
 config statistics 'collectd_interface'
 	option enable '1'
@@ -209,9 +210,9 @@ config statistics 'collectd_tcpconns'
 	option AllPortsSummary '0'
 
 config statistics 'collectd_thermal'
-	option enable '0'
-	option IgnoreSelected '0'
-	option Device ''
+	option enable '1'
+	option IgnoreSelected '1'
+	option Device 'cooling_device0 cooling_device1'
 
 config statistics 'collectd_uptime'
 	option enable '0'
diff --git a/modules/luci-base/root/etc/config/luci b/modules/luci-base/root/etc/config/luci
index 46947ae4..f922d3d4 100644
--- a/modules/luci-base/root/etc/config/luci
+++ b/modules/luci-base/root/etc/config/luci
@@ -23,9 +23,12 @@ config internal ccache
 	option enable 1
 		
 config internal themes
+	option Bootstrap '/luci-static/bootstrap'
 
 config internal apply
 	option rollback 90
 	option holdoff 4
 	option timeout 5
 	option display 1.5
+
+
diff --git a/themes/luci-theme-openwrt-2020/root/etc/uci-defaults/30_luci-theme-openwrt-2020 b/themes/luci-theme-openwrt-2020/root/etc/uci-defaults/30_luci-theme-openwrt-2020
index 7c49acfd..07d87b39 100755
--- a/themes/luci-theme-openwrt-2020/root/etc/uci-defaults/30_luci-theme-openwrt-2020
+++ b/themes/luci-theme-openwrt-2020/root/etc/uci-defaults/30_luci-theme-openwrt-2020
@@ -4,7 +4,7 @@ if [ "$PKG_UPGRADE" != 1 ]; then
 	uci get luci.themes.OpenWrt2020 >/dev/null 2>&1 || \
 	uci batch <<-EOF
 		set luci.themes.OpenWrt2020=/luci-static/openwrt2020
-		set luci.main.mediaurlbase=/luci-static/openwrt2020
+#		set luci.main.mediaurlbase=/luci-static/openwrt2020
 		commit luci
 	EOF
 fi
